webapp/portlet/src/main/webapp/space-menu/main.js

```diff
@@ -36,4 +39,13 @@ export function init(settings) {
     template: `<space-menu v-cacheable="{cacheId: '${cacheId}'}" id="${appId}" :navigations="navigations" :selected-navigation-uri="selectedNavigationUri" />`,
     vuetify,
   }).$mount(appElement);
+
+  new Vue({
+    mounted() {
```

Comment:
I didn't get it, could you please explain why this change ? and what does it affect, especially why do we need to define anew app in the same main.js file ? (This is the first I encountered such use case in fact)
@pnedonosko

Reply:
Thank you @boubaker or looking at the code. But please clarify the question. It's not clear what does the _change_ 

> could you please explain why this change ?

As for main.js, it's part of a place we found good for adding an extension point to deal with Social space title. Its actual content in `bannerLogo.jsp`. And two instances of Vue, what is wrong with this? Can you tell how you would do this better?

Reply:
I guess, the question is clear. But it seems that we lack of best practices here.
In fact the description didn't described well, why change the existing code, is it a bu, is it an improvement. If it's an improvement for what and why is it the best solution. Here are some guidelines  : http://developer.exoplatform.org/#id-commits-messages-best-practices
After that, as any developer looking at PRs and who can comment/approve/disapprove code (not only TL of FT BTW ;-) ), I personally developed a lot of Vue applications and the solution I found here (of what problem?) seems not coherent to what we do in product usually. So my question is simple, why this change and to do what ?